include/tables/actor_table.h

```diff
@@ -11,474 +11,474 @@
  *
  * DEFINE_ACTOR_UNSET is needed to define empty entries from the original game
  */
-/* 0x0000 */ DEFINE_ACTOR_INTERNAL(Player, ACTOR_PLAYER, ALLOCTYPE_NORMAL, "Player")
+/* 0x0000 */ DEFINE_ACTOR_INTERNAL(Player, ACTOR_PLAYER, ACTOR_ALLOC_NORMAL, "Player")
```

Comment:
Why not drop the ACTOR_ALLOC_ part in this table? It can be inserted with macros, and it's not like this cooperates properly with IDEs anyway.

Reply:
because it breaks grepability (can't grep / ctrl shift F it) :/

Reply:
You could say the same thing for leaving `ovl` off the names.

Reply:
yeah I do think it would be nice to properly refer the segments

but that's not happening as we would then need another argument for the actorinit symbol name (or rename the actorinit symbols to use the overlay segment name), and I feel like that wouldn't have much support?

Reply:
I do think that ACTOR_ being the prefix poses the same problem as the scene draw configs from a little while ago-- it looks like an actor id enum
would prefer ACTORALLOC_ or something, idk

Reply:
`ACTOROVL_ALLOC_` ? (reminder this deals with overlays not the actor instances)

Reply:
yea seems good